packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/controller.js

```diff
@@ -0,0 +1,80 @@
+const {
+	getLPAQuestionnaireByAppealId,
+	createLPAQuestionnaire,
+	patchLPAQuestionnaireByAppealId
+} = require('./service');
+const logger = require('#lib/logger');
+const ApiError = require('#errors/apiError');
+
```

Comment:
I wonder if there's a way to dry this file up a bit but I might be overfactoring